pom.xml

```diff
@@ -51,6 +51,17 @@
             <artifactId>github</artifactId>
             <version>1.14.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
+            <version>0.2-beta-6</version>
```

Comment:
:bug: this plugin should not depend on `branch-api`. Perhaps need to introduce a `SCMNavigatorOwner` that `OrganizationFolder` could implement?